### PR TITLE
fix deploy template

### PR DESCRIPTION
**Pull request by julianduque**
Adding the template to use the `main` branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ Backend for the [Emote Widget](https://github.com/fostive/emote-widget)
 
 ## Installation & Deployment
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/fostive/emote-widget/tree/main)
```
Comment (crcastle) [inline README.md:11]: I think this should be `emote-server` instead of `emote-widget`?

**Review by crcastle — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (julianduque) [reply README.md:11]: Oh you are right!!

Comment (julianduque) [reply README.md]: @crcastle done!

**Comment by crcastle** [comment]
lgtm!

**Review by crcastle — APPROVED** [review]

**Pull request merged**